### PR TITLE
:heart_eyes: don't use the 'change' trigger to save to the local storage

### PR DESCRIPTION
**Pull request by GabbasovDinar**
If an order is changed every time this PR fixes an issue with rerendering of the product list. It's related to new changes in Odoo 11.0 https://github.com/odoo/odoo/blame/11.0/addons/point_of_sale/static/src/js/screens.js#L846

### CODE REVIEW

**Comment by GabbasovDinar** [comment]
@yelizariev Can you check it?

**Review by yelizariev — CHANGES_REQUESTED** [review]
Can there be other reason to use ``trigger('change')``?

Anyway, you have to update change log and *Tested on* note in README

**Comment by GabbasovDinar** [comment]
@yelizariev Who can test these changes? 
image

File: pos_multi_session/doc/changelog.rst [hunk 1]
```diff
@@ -1,6 +1,9 @@
-`4.2.2`
+`4.2.3`
 -------
+- **FIX:** Rerendering the Products list after change a current order
```
Comment (yelizariev) [inline pos_multi_session/doc/changelog.rst:3]: What is the problem with rerendering?

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: pos_multi_session/doc/changelog.rst (hunk 1 above)
Comment (GabbasovDinar) [reply pos_multi_session/doc/changelog.rst:3]: @yelizariev  If we have a rerender of the product list every time, then we will get a hang and slow response of the system. 

How can I change the log message?

**Comment by GabbasovDinar** [comment]
I will make cherry-pick of a commit from https://github.com/it-projects-llc/pos-addons/pull/809 after merging it

**Pull request closed without merging**